src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -116,9 +116,9 @@ ul.dock {
       padding: 5px 0px;
       .border-top-radius(5px);
       .border-bottom-radius(5px);
-      background-color: @bg-dock-menu-color;
+      background-color: @mno-bg-settings-dock-menu;
 
-      .box-shadow(@box-dock-shadow);
+      .box-shadow(@mno-box-dock-shadow);
```

Comment:
The name of this variable should be: `@mno-dock-box-shadow`.
The dock variables must respect the convention `@mno-dock-*`.